Example/GeneralPreferenceViewController.swift

```diff
@@ -3,7 +3,8 @@ import Preferences
 
 final class GeneralPreferenceViewController: NSViewController, Preferenceable {
 	let toolbarItemTitle = "General"
-	let toolbarItemIcon = NSImage(named: NSImage.preferencesGeneralName)!
+	let toolbarItemIcon: NSImage? = NSImage(named: NSImage.preferencesGeneralName)!
+    let toolbarItemIdentifier: NSToolbarItem.Identifier = .preferenceGeneral
```

Comment:
Since almost everyone would have a `General` preference tab. I wonder if we should ship with a `PreferenableGeneral` type thing that has all these fields already filled out. My goal is to require the least amount of boilerplate possible to set up a preferences window.

Reply:
I think that's a fair point, providing a common set of preferences (General, Advanced, Update, maybe License) -- what do you think about collecting this in a new issue so we can do it in one swoop, solving the localization issue #12  ?

Reply:
Sure

Reply:
see #13 👍